flycheck.el

```diff
@@ -6644,7 +6644,25 @@ See URL `http://golang.org/cmd/go'."
           (optional column ":") " "
           (message (one-or-more not-newline)
                    (zero-or-more "\n\t" (one-or-more not-newline)))
-          line-end))
+          line-end)
+   (info line-start
+         (message "can't load package: package "
+                  (one-or-more (not (any ?: ?\n)))
+                  ": found packages "
+                  (one-or-more not-newline))
+         line-end))
+  :error-filter
+  (lambda (errors)
+    (dolist (error errors errors)
```

Comment:
Please return `errors` explicitly after the loop.  I am no particular fan of this behaviour of `dolist`.